.ci/windows.groovy

```diff
@@ -91,7 +94,8 @@ pipeline {
           when {
             beforeAgent true
             expression {
-              return env.BUILD_ELASTIC_AGENT_XPACK != "false" && params.windowsTest
+              return params.windowsTest
+              //return env.BUILD_ELASTIC_AGENT_XPACK != "false" && params.windowsTest
```

Comment:
Did you intend to leave this comment (and those below) in?

Reply:
yes